app/models/course/assessment/category.rb

```diff
@@ -34,13 +36,15 @@ def other_categories_remaining?
   private
 
   def build_initial_tab
-    tabs.build(title: Course::Assessment::Tab.human_attribute_name('title.default'),
-               weight: 0, category: self)
+    if tabs.empty?
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.